docs/library/axi_ad9963/index.rst

```diff
@@ -248,15 +248,15 @@ Software Support
 --------------------------------------------------------------------------------
 
 The software for this part can be found as part of the :adi:`ADALM2000`
-(or shortly, M2K) reference design. It is supported by both :git-linux:`/` and
-:git-no-os:`/`.
+(or shortly, M2K) reference design. 
+
+* Linux device driver at :git-linux:`drivers/iio/adc/ad9963.c`
 
 References
 -------------------------------------------------------------------------------
 
 * :adi:`AD9963`
 * HDL source code at :git-hdl:`library/axi_ad9963`
```

Comment:
```suggestion
* HDL IP core at :git-hdl:`library/axi_ad9963`
* HDL project at :git-hdl:`projects/m2k`
* HDL project documentation at :ref:`m2k`
```
Specify the HDL project (projects/m2k) and the project doc (although this is not published yet, but not to come back and edit this page as well).